### PR TITLE
added snackbar component

### PR DESCRIPTION
**Pull request by pariyar07**
3 types of snackbar added.

### CODE REVIEW

File: styles/snackbar.css [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+    border: none;
+    background-color: white;
+    color: rgb(0, 0, 0);
+    padding: 0.5rem;
+    border-radius: 5px;
+    transition: 0.5s;
+}
+.button-container>button:hover{
+    background-color: rgb(100, 100, 100);
+    color: rgb(255, 255, 255);
```
Comment (animeshgarai09) [inline styles/snackbar.css:29]: Any specific reason for using colors directly rather than using them with variables?

**Review by animeshgarai09 — APPROVED** [review]
All good.

**Comment by yogesh-** [comment]
> 3 types of snackbar added.

Always add image or video of the screen/items/components you are adding.

File: styles/snackbar.css (hunk 1 above)
Comment (yogesh-) [reply styles/snackbar.css:29]: use  color variables

**Review by yogesh- — APPROVED** [review]
looks ok

**Review by psychoactiv — APPROVED** [review]
Good to go

**Pull request merged**